scripts/get-release-tag-str.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script derives a version string from the version fields in a target Go
+# file. It also checks if the derived version string is SemVer compliant.
+
+# get_git_tag_version_string reads the version fields from the given file and
+# constructs a git tag version string.
+get_git_tag_version_string() {
```

Comment:
Doesn't bash or the command line only really know strings? IMO the "_string" suffix of the function and the "-str" suffix of the file are a bit redundant.